### PR TITLE
Save and retrieve UUID based on MAC address

### PR DESCRIPTION
**Pull request by eliseomartelli**
Save and retrieve from file system UUID based on MAC address, removes hardcoded UUID for HomeKit

### CODE REVIEW

File: homekit/uuidgen.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/google/uuid"
+)
+
+func GetUUID(deviceName string) uuid.UUID {
+	uuidStore := fmt.Sprintf("./.uuid%s", deviceName)
```
Comment (AlbanSeurat) [inline homekit/uuidgen.go:11]: Could you use « %s/uuid.cfg » as there is already a directory with device information

File: homekit/accessory.go [hunk 2]
```diff
@@ -39,9 +41,9 @@ func NewAccessory(deviceId string, features globals.Features) *Accessory {
 		protovers: "1.1",
 		acl:       "0",
 		rsf:       "0x0",
-		Pi:        UUID{uuid.MustParse("c9f33810-4c86-4651-96f9-7531b79cb410")}, // should be unique by user (generated and saved)
-		Gid:       UUID{uuid.MustParse("c9f33810-4c86-4651-96f9-7531b79cb410")}, // should be unique by user (generated and saved)
-		Psi:       UUID{uuid.MustParse("c9f33810-4c86-4651-96f9-7531b79cb410")}, // should be unique by user (generated and saved)
+		Pi:        UUID{currentUuid}, // should be unique by user (generated and saved)
+		Gid:       UUID{currentUuid}, // should be unique by user (generated and saved)
```
Comment (AlbanSeurat) [inline homekit/accessory.go:45]: You can remove the comments

**Comment by eliseomartelli** [comment]
Removed comments and changed save file path as reqeusted @AlbanSeurat

**Comment by eliseomartelli** [comment]
Oops, deviceId and device name are similar and the directory is not matching. I'll look into it later

**Comment by eliseomartelli** [comment]
🎉 it works now ahah

**Comment by AlbanSeurat** [comment]
Thanks

**Pull request merged**